### ISSUE TITLE
missing data

### ISSUE THREAD

**Issue by leticiapr** [visualize-admin/visualization-tool#1309]
The data is not displayed and no diagrams are shown. Only an error message is displayed

**To Reproduce**

1. open this link:  https://int.visualize.admin.ch/v/EbXixKlDKslp?dataSource=Int 
2. Here you can see the error message 

**Expected behavior**
normally the data should be displayed

**Screenshots or video**

![image](https://github.com/visualize-admin/visualization-tool/assets/157011391/866fe677-de77-43ec-8ce4-6aae6f277866)

Visualize environment and version: [e.g. INT v3.25.0]
Microsoft Edge:  Version 120.0.2210.121 (Offizielles Build) (64-Bit)

**Comment by ortnever** [comment]
I mean the chart is correct but Visualize do not have to propose this date (2014) for this Location (St-Martin) because there is no value for this combination.

**Comment by sosiology** [comment]
Hi @ortnever ,

Thank you for your feedback and the valuable input relating to the use case where there is data for the current year, but with no defined measure values. 
While this issue is mentioned in the context of most recent date filter feature, it's also easily reachable by the editor by just manually changing the year to 2014. We believe that in order to handle this situation properly, we should also disable a possibility to select such filter in "regular" mode, or even better – indicate that the values are there, but are undefined. 
Currently, we only highlight undefined values in certain charts types (gray areas in maps), but don't do so for other chart types (e.g. bar charts).
We think that generally, undefined values "tell" us something – in the example above there are values in the table, but measure values happen to be undefined –  in this case it could mean that there were some measurements done in 2014, but not of the Schwermetallkonzentration measure, otherwise these rows shouldn't appear in the dataset (if all measures were undefined).
To sum up, we could either try to disable selecting of such filter combinations that yield only undefined values, inform the user with a banner of top that no values were found or handle undefined values in a better way across all chart types in a next iteration.

**Comment by ortnever** [comment]
Hi @Solange : Vielen dank für die Erklärungen. Wie am Weekly Call vom 6.02 abgemacht, lassen wir das in einem ersten Schritt so.
